src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('/src/images/star.svg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--1 .stars__star:nth-child(-n + 1),
+  &--2 .stars__star:nth-child(-n + 2),
+  &--3 .stars__star:nth-child(-n + 3),
+  &--4 .stars__star:nth-child(-n + 4),
+  &--5 .stars__star:nth-child(-n + 5) {
+    background-image: url('/src/images/star-active.svg');
```

Comment:
Similar to the previous comment, ensure that the path to the 'star-active.svg' is relative to the compiled CSS file. Adjust the path if necessary, taking into account where the CSS file will be located after the build process.